### ISSUE TITLE
Show search terms on search results page

### ISSUE THREAD

**Issue by danielnaab** [18F/cv_faq#766]
## Description
As a user viewing search results, I would like to clearly see the search terms I entered in the search results, to help me maintain context.

## Notes
We tried to prefill the search box input with the current search terms as part of #736, but encountered some issues with the behavior of `accessible-autocomplete`. This PR, when merged, should address the problem: https://github.com/alphagov/accessible-autocomplete/pull/425

## Acceptance Criteria
- [ ] Search query visible in search box when displaying results

## Tasks
- [ ] PO approved
- [ ] Usability tested

## Definition of Done
- [ ] Pull requests meet technical definition of done
- [ ] Code is properly styled per USWDS
- [ ] Finished design has been compared against mockup
- [ ] Documentation is complete
- [ ] CircleCI tests have passed
- [ ] Automated accessibility scans have passed
- [ ] Manual accessibility testing completed
- [ ] Code has been successfully built and deployed into a staging (dev) environment

**Comment by brandocalrissian** [comment]
Blocked on bug w/autocomplete widget (see notes above)

**Comment by brandocalrissian** [comment]
related to #737

**Comment by brandocalrissian** [comment]
Same bug is here.

**Comment by brandocalrissian** [comment]
Don't do anything on this until a proposal for changing search is denied. Only relevant if staying on search.gov.

**Comment by brandocalrissian** [comment]
Project wrapped up. No longer needed.
